CruGlobal/mpdx-react
# MPDX-8429 Fix 502 error on 14-month report

## Description
The 14-month Proxy GraphQL query always failed on `iryna.oleksiuk@cru.org`'s account. This is due to the amount of data being returned, and the server couldn't cope.

For a very quick fix, I have called the API directly from the frontend, instead of using Next.js backend. I believe this will fix the issue for now, but the main issue is that we need to make this query into a proper GraphQL query.

I hope this doesn't cause the query to be slow for others. - In my testing, it has somewhat sped it up.

## To Test:
1. Impersonate `iryna.oleksiuk@cru.org` and see if the page returns a 502.
2. Impersonate others or use your own account to ensure the report is working as it should.


Slack Message: https://cru-main.slack.com/archives/GG47BDJDU/p1730814260125489
Jira Ticket: https://jira.cru.org/browse/MPDX-8429

## Checklist:
